src/test/suite/index.ts

```diff
@@ -14,8 +14,8 @@ export async function run(): Promise<void> {
         bail: false,
         slow: 20000,
         fullTrace: true,
-        retries: 3,
         grep: ".*",
+        retries: 0,
```

Comment:
This is tricky. I added retries last year because I wanted green CI, but the problem is that many bugs relating to this project are timing related and retrying with CI could hide some reliability regressions. A test that fails 2/3 times is still a failing test. For now, I removed the retries just so I could really examine which tests were flaky and if I could fix them, but undecided if I will keep them.